File: programs/python/recursion/output_programs/recursive_factorial_2_errors_marked.py

def recur_factorial ( n  <span style="color:red">)</span> : 
<br/> 	if n == 1 
<br/> 		return n 
<br/> 	else : 
<br/> 		return n * recur_factorial ( n - 1  <span style="color:red">)</span>  <span style="color:red">)</span> 
<br/> num = 7 
<br/> if num < 0 : 
<br/> 	print ( "Sorry, factorial does not exist for negative numbers" ) 
<br/> elif num == 0 : 
<br/> 	print "The factorial of 0 is 1"  <span style="color:red">)</span>  <span style="color:red">`</span> 
<br/> else : 
<br/> 	print ( "The factorial of" , num  <span style="color:red">,</span>  <span style="color:red">,</span> "is" , recur_factorial ( num ) ) 
<br/>  <span style="color:red">@</span> 

File: programs/c/recursion/output_programs/recursive_factorial_2_errors_marked.c

int main ( ) { 
int n ; 
printf ( "Enter a positive integer: " ) ; 
scanf (  <span style="color:red">"%d"</span> , & n ) ; 
printf ( "Factorial of %d = %ld" , n , multiplyNumbers ( n ) ) ; 
return 0 ; 
} 
long int multiplyNumbers ( int n ) { 
if ( n >= 1 ) { 
return n multiplyNumbers ( n - 1 )  <span style="color:red">!</span> ; 
} 
else { 
return 1  <span style="color:red">^</span> ; 
} 
} 